### PR TITLE
Stuart to maintainer

### PR DESCRIPTION
**Pull request by SimonBiggs**
Now's possibly a good time to add employer details if so desired. Up to you.

### CODE REVIEW

**Review by sjswerdloff — APPROVED** [review]
if underline is important, please fix my update.
Sorry for any hassle.

File: README.rst [hunk 1]
```diff
@@ -178,19 +178,21 @@ Past contributors
 .. |hgugm| image:: https://github.com/pymedphys/pymedphys/raw/master/docs/logos/HGUGM_200x200.png
     :target: `Hospital General Universitario Gregorio Marañón`_
 
-.. _`Riverina Cancer Care Centre`: http://www.riverinacancercare.com.au/
+.. _`Riverina Cancer Care Centre`: https://www.riverinacancercare.com.au/
+
+.. _`ELEKTA Pty Ltd`: https://www.elekta.com/
```
Comment (SimonBiggs) [inline README.rst:183]: Key to the underscore is it's needed to support this. Is this link okay?

**Comment by SimonBiggs** [comment]
> Sorry for any hassle.

No hassle at all. Exciting :)

**Review by sjswerdloff — APPROVED** [review]
good to go

**Comment by SimonBiggs** [comment]
🍻 🍻

**Pull request merged**